operatorconfig/driverconfig/powerflex/v2.12.0/upgrade-path.yaml

```diff
@@ -0,0 +1,3 @@
+
+ minUpgradePath: v2.11.0
```

Comment:
can you change the `minUpgradePath` to supported n-2 versions of driver and modules.

Reply:
addressed